amazons3/amazons3_connector.bal

```diff
@@ -267,3 +238,10 @@ function getClientEndpoint(string bucketName) returns http:Client {
     clientEndpoint.init(clientConfig);
     return clientEndpoint;
 }
+
+function setResponseError(int statusCode, xml xmlResponse) returns error {
```

Comment:
Move this method to the file where we write other utils methods

Reply:
Fixed.